src/App.tsx

```diff
@@ -14,11 +16,114 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+const App: React.FC = () => {
+  const [start, setStart] = useState(false);
+  const [revers, setRevers] = useState(false);
+  const [typeSort, setTypeSort] = useState('none');
+  const [fildSize, setFildSize] = useState(goodsFromServer.length);
+
+  function reset() {
+    setRevers(false);
+    setTypeSort('none');
+    setFildSize(goodsFromServer.length);
+  }
+
+  return (
+    <div className="App bg-info">
+      <div className="App-wrap">
+        {!start && (
+          <button className="btn bg-primary btn-my" type="button" onClick={() => setStart(true)}>Start</button>
```

Comment:
```suggestion
          <button
            className="btn bg-primary btn-my"
            type="button"
            onClick={() => setStart(true)}
          >
            Start
          </button>
```

Reply:
I corrected the errors, but now I can't request a check again